### ISSUE TITLE
You have to uninstall Elpis to install the new version.

### ISSUE THREAD

**Issue by BrianDaTruth** [adammhaile/Elpis#178]
Title.

Says there is already a version of Elpis already installed on the computer.

**Comment by Javier-Fuentes** [comment]
To install the newer version just uninstall elpis from the control center hope that helps you out

**Comment by Wires77** [comment]
Did you get Elpis from GitHub or from the built-in updater?  The updater should upgrade Elpis in-place.  I've seen it not quit Elpis.exe before and you have to force quit through the task manager, however.

**Comment by BrianDaTruth** [comment]
@Javier-Fuentes Yes, I did this, but I just wanted to report the bug.

@Wires77 I tried the auto update at first, and it failed. I then came to the GitHub to get the latest .exe it also failed. I uninstalled it Elpis and installed latest .exe from GitHub. 

This is a picture of the error I got when it tried to auto update.
http://i.imgur.com/b2z3vWC.png

**Comment by Wires77** [comment]
What version of Elpis were you updating from and what OS version?  If we can reproduce it maybe I can get some logs from you and take a closer look.

**Comment by BrianDaTruth** [comment]
Windows 10 and 1.5.2
I'm happy to help you out. I've been using Elpis for a long time and really like it.

**Comment by Wires77** [comment]
Huh, I updated fine earlier, but now I can reproduce this after downloading it fresh.  I'll try and find a fix for this before the full release.  Thanks for bringing it up!

**Comment by Wires77** [comment]
Okay, it seems like the installer isn't updating the version when I build, so I had to manually change that and generate a new ProductCode.  I'm skeptical of my build setup, since I had to install a plug-in for VS2013 since the Windows Installer projects aren't natively supported after 2010.  Oddly, running the install from Visual Studio works fine, but not from the file.

@adammhaile or @seliver I'm assuming neither of you experienced this and it's an issue with my build environment, but if you'd like to check if that works as well.

@BrianDaTruth I'll upload the updated installer, so if you want to try upgrading from 1.5.2 through Elpis we can see if that works.
